WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -655,16 +670,13 @@ def setJobAttributes( self, jobID, attrNames, attrValues, update = False, myDate
     if len( attr ) == 0:
       return S_ERROR( 'JobDB.setAttributes: Nothing to do' )
 
-    cmd = 'UPDATE Jobs SET %s WHERE JobID=%s' % ( ', '.join( attr ), jobID )
+    cmd = 'UPDATE Jobs SET %s WHERE JobID in ( %s )' % ( ', '.join( attr ), ', '.join( jIDList ) )
```

Comment:
You could use the utilities in Core/Utilities/List

Reply:
I could. I guess you mean intListToString/stringListToString. These are one-liner functions which do exactly the same but it will be one more redirection which does not help readers. The used syntax is very clear itself. Also, these functions are actually not clearly defined. The difference is only whether you want or not to put quotes around the list members. So, if I want to achieve the same effect as in the original code I should have done: intListToString( attr ). This is a real mess. I would say that these functions are quite counter-productive and should not be scrapped.

Reply:
Oh, typing... I meant "should now be scrapped" :)

Reply:
Well, actually this is used quite a lot, in the DataManagement , in particular. So, no scrapping. But as a rule of thumb ( can be formalized ? ) we should not use one evident line utility functions

Reply:
I 100% agree with you. However this one I would say is not so straight forward ! Especially that I believe it should be modified to include the sql escaping !

Reply:
This is another story, and as far as I can see, the SQL input elements are all escaped here.

Reply:
Why do you say it is another story ? 
They might be escaped  here (I do not know to be honest :-) ), but I still believe that there should be a utility function, doing exactly this + the escaping. Simon had to do it no later than this week :-)

Reply:
I agree that a set of utility functions to escape vectors of values would be useful. Just that this is not subject of this PR.